include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1601,6 +1601,12 @@ class MatrixFree : public Subscriptor
   bool
   mapping_initialized() const;
 
+  /**
+   * Return the level of the mesh to be worked on.
+   */
+  unsigned int
+  get_level_mg_handler() const;
```

Comment:
I'll agree with @tjhei that that is an unfortunate naming -- the function doesn't return a handler, but a level.